packages/kbn-apm-synthtrace/README.md

```diff
@@ -154,7 +154,7 @@ Note:
 Run the Jest tests:
 
 ```
-node scripts/jest --config ./packages/kbn-apm-synthtrace/jest.config.js
+node scripts/jest --config ./packages/kbn-apm-synthtrace/jest.config.js [--updateSnapshot]
```

Comment:
I don't think we should run this automatically. It will mask unintended changes.

Reply:
It's not meant to be run automatically, just information in case you need to update snapshots. In fact if you try to run this you'll get an error

Reply:
Yeah, I think jest will tell you when snapshots are failing. IMHO it isn't different from any other failing test, we should inspect the failure and then either update snapshots accordingly or fix the underlying issue if the change was unexpected.